test/suites/backup.sh

```diff
@@ -260,10 +260,10 @@ test_backup_import_with_project() {
   lxc profile device remove default root
 
   # This should fail as the expected storage is not available, and there is no default
-  ! lxc import "${LXD_DIR}/c3.tar.gz"
+  ! lxc import "${LXD_DIR}/c3.tar.gz" || false
```

Comment:
@monstermunchkin could you update the commit msg here to explain this change. Thanks

Reply:
Shellcheck complains about `! <some-command>`, and suggests it be changed to `! <some-command> || false`. We do have this kind of code in other places in our test suite.
I don't really care much about this but shellcheck keeps bothering me every time I checkout a clean branch and run static analysis.

Reply:
@monstermunchkin cool, good to know, I thought it may be a functional change. :)